web/app/themes/ppj/partials/mobileNavigation.php

```diff
@@ -32,7 +32,13 @@
 <div class="mobile-nav">
     <div class="mobile-nav__overlay"></div>
     <button class="mobile-nav__close-button"
+            aria-label="Close mobile mobile navigation menu button"
             onclick="ppj.closeNavMenu()">
+        <span class="mobile-nav__close-button-text">Close mobile menu</span>
+        <img class="mobile-nav__close-button-image"
+             src="/app/themes/ppj/dest/img/svg/close.svg"
+             alt="Close mobile mobile navigation menu button image"
```

Comment:
Same here – 'mobile' is repeated twice. And I'm not convinced 'mobile' needs to be used at all anyway.

Reply:
Same as above.